src/components/ParticipantList.vue

```diff
@@ -251,32 +250,21 @@ Licensed under the Elastic License 2.0. */
   }
 
   const createParticipant = (amount: number): void => {
-    const i = amount || 1;
     const newParticipants: Participant[] = [];
-    // starWarsNames.all.length = 93
-    const openNames: string[] = starWarsNames.all.filter(
-      (n) => !participantsList.value.find((p) => p.alias === n),
-    );
 
-    const addNew = (alias: string): number =>
+    const maxId = participantsList.value.reduce(
```

Comment:
das ist so unglaublich komplizierter code: 

```
const maxId = Math.max(
  0,
  ...participantsList.value.map(p => p?.participantId ?? 0)
);
```

sollte es ja auch tun.

Reply:
Nice usage of the spread operator. Done in https://github.com/MORE-Platform/more-studymanager-frontend/pull/591/commits/c83c6ea2d632fa223cdd0964ca5200f338a242b4